proxy/reverse.go

```diff
@@ -118,6 +121,9 @@ func (p *reverseProxy) ServeHTTP(rw http.ResponseWriter, clientreq *http.Request
 		redirectRequest(proxyreq, ep.URL)
 
 		res, err = p.transport.RoundTrip(proxyreq)
+		if requestCanceled {
```

Comment:
canceling a request simple closes the underlying connection and you will get an error that says "use of closed network connection" and I'm not sure if we can detect if we closed it our self or the remote server closed it.